src/slm/thread/slmthread.cpp

```diff
@@ -253,8 +253,8 @@ main(int argc, char* argv[])
 
     bool usingLogPr = slm.isUseLogPr();
 
-    #define EffectivePr(a)  (usingLogPr ? ((a) / log(2.0)) : -log2f((a)))
-    #define OriginalPr(b)   (usingLogPr ? ((b) * log(2.0)) : exp2(-(b)))
+    #define EffectivePr(a)  (usingLogPr ? ((a) / log(2.0f)) : -log2f((a)))
```

Comment:
maybe we can be more consistent here, say, just use the STL overridden variant:
```c++
#define EffectivePr(a)  (usingLogPr ? ((a) / log(2.0f)) : -log2((a)))
#define OriginalPr(b)   (usingLogPr ? ((b) * log(2.0f)) : exp2(-(b)))
```

Reply:
see also https://github.com/sunpinyin/sunpinyin/pull/77/commits/08074291801f7b7d25501c9275f25c7474dfbe0e